pkg/controller/controller_utils_test.go

```diff
@@ -376,6 +376,24 @@ func TestDeletePodsAllowsMissing(t *testing.T) {
 	assert.True(t, apierrors.IsNotFound(err))
 }
 
+func TestIsTerminatingPods(t *testing.T) {
```

Comment:
As above, I would suggest not moving this function, then the test isn't needed. If the test was to go in, then I would suggest using named test cases (e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/job_controller_test.go#L275-L313), rather than a list expectations. The names of test cases can be descriptive of the scenario.

Reply:
updated.